Update 8.0.32-25.0.md (#104)
percona/pxb-docs: docs/release-notes/8.0/8.0.32-25.0.md

## Release highlights

> This release merges the MySQL 8.0.32 code base. Percona has implemented a two-release process for each version. The first release merges the code base of the latest MySQL version. This release helps those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains bug fixes and any improvements or features.

Reviewer: ```suggestion
This release merges the MySQL 8.0.32 code base. Percona Xtrabackup has implemented a two-release process for each version. The first release merges the code base of the latest MySQL version. This release helps those customers who need an updated Percona XtraBackup that aligns with the latest MySQL Server version. The second release contains bug fixes and any improvements or features.
```

Author: Done.
